src/app/external-cluster-wizard/steps/external-cluster/provider/gke/component.ts

```diff
@@ -160,6 +174,9 @@ export class GKEClusterSettingsComponent
     this.form.valueChanges.pipe(takeUntil(this._unsubscribe)).subscribe(_ => {
       this._updateExternalClusterModel();
       this._updateExternalMachineDeployment();
+      if (this.controlValue(Controls.Zone)?.main) {
```

Comment:
When Release Channel is removed.

Remove previous version 

https://user-images.githubusercontent.com/17727069/182349016-25036ef8-b459-4434-a029-914197c53354.mp4